Views/guiV1.py

```diff
@@ -1,10 +1,515 @@
+        self.config(menu=menu_bar)
+        # File
+        file_menu = Menu(menu_bar, tearoff=0)
+        file_menu.add_command(label="New", command=self.new_file)
+        file_menu.add_command(label="Open...", command=self.open_file)
+        file_menu.add_command(label="Save", command=self.save_file)
+        file_menu.add_separator()
+        file_menu.add_command(label="Exit", command=self.quit)
+        menu_bar.add_cascade(label="File", menu=file_menu)
+
+        # Edit
+        edit_menu = Menu(menu_bar, tearoff=0)
+        edit_menu.add_command(label="Undo", command=self.undo_action)
+        menu_bar.add_cascade(label="Edit", menu=edit_menu)
+
+        # Classes
+        classes_menu = Menu(menu_bar, tearoff=0)
+        classes_menu.add_command(label="New", command=self.add_class)
+        classes_menu.add_command(label="Delete", command=self.delete_class)
+        classes_menu.add_command(label="Rename", command=self.rename_class)
+        classes_menu.add_command(label="List Attributes", command=self.list_attributes)
+        classes_menu.add_command(label="List Classes", command=self.list_classes)
+        menu_bar.add_cascade(label="Classes", menu=classes_menu)
+
+        # Relationships
+        relationships_menu = Menu(menu_bar, tearoff=0)
+        relationships_menu.add_command(label="Add", command=self.add_relationship)
+        relationships_menu.add_command(label="Delete", command=self.delete_relationship)
+        relationships_menu.add_command(label="List", command=self.list_relationship)
+        menu_bar.add_cascade(label="Relationships", menu=relationships_menu)
+
+        # Attributes
+        attributes_menu = Menu(menu_bar, tearoff=0)
+        attributes_menu.add_command(label="New Field", command=self.add_field)
+        attributes_menu.add_command(label="Delete Field", command=self.delete_field)
+        attributes_menu.add_command(label="Rename Field", command=self.rename_field)
+        attributes_menu.add_separator()
+        attributes_menu.add_command(label="New Parameter", command=self.new_param)
+        attributes_menu.add_command(label="Delete Parameter", command=self.delete_param)
+        attributes_menu.add_command(label="Rename Parameter", command=self.rename_param)
+        attributes_menu.add_separator()
+        attributes_menu.add_command(label="New Method", command=self.new_method)
+        attributes_menu.add_command(label="Delete Method", command=self.delete_method)
+        attributes_menu.add_command(label="Rename Method", command=self.rename_method)
+        menu_bar.add_cascade(label="Attributes", menu=attributes_menu)
+
+        # Help
+        help_menu = Menu(menu_bar, tearoff=0)
+        help_menu.add_command(label="Classes", command=self.helpClasses)
+        help_menu.add_command(label="Relationships", command=self.helpRelationships)
+        help_menu.add_command(label="Attributes", command=self.helpAttributes)
+        menu_bar.add_cascade(label="Help", menu=help_menu)
```

Comment:
What does the help menu do in the gui? Does it need to be handled differently because the user isn't putting in commands?